general/g.tempfile/g.tempfile.html

```diff
@@ -10,10 +10,10 @@ <h2>DESCRIPTION</h2>
 
 <p>
 <em>g.tempfile</em>
-creates an unique file and prints the name. The user is required to provide
-a process-id which will be used as part of the name of the file.
```

Comment:
While at it, can you confirm that the tmp dir for mapset is per node? Just note caveats of using only PID (recent discussions in https://github.com/OSGeo/grass-addons/pull/1200#discussion_r1766046582 and https://github.com/OSGeo/grass-addons/pull/1202#discussion_r1765505630). The per-mapset tmp dir is trying to address there I think, but I don't see that here in the doc.

Reply:
Yes, the path returned by `g.tempfile` does include the hostname.
The base directory is `$LOCATION/$MAPSET/.tmp/$HOSTNAME/`. Then the file- (or directory) name is _PID.IDX_, where IDX is a 0-indexed number of calls of g.tempfile with the same PID.


That said, I think the manual can benefit from including the essence of the discussions in the referenced PRs and the new style guide...